docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    kept = conn.create_table("kept", kept)
</pre><p>Since you're using an in-memory connection, <code>.cache()</code> would be a good choice here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>